django_celery_monitor/managers.py

```diff
@@ -39,3 +92,23 @@ def purge(self):
                 'DELETE FROM {0.db_table} WHERE hidden=%s'.format(meta),
```

Comment:
I was, too. This is purely for historical reasons, coming over from django-celery. Will fix.